llvm/test/Transforms/PGOProfile/memprof_internal_linkage.ll

```diff
@@ -12,7 +12,8 @@
 ; RUN: opt < %s -passes='memprof-use<profile-filename=%t.memprofdata>' -S | FileCheck %s
 
 ; CHECK: call {{.*}} @_Znam{{.*}} #[[ATTR:[0-9]+]]
-; CHECK: attributes #[[ATTR]] = { builtin allocsize(0) "memprof"="notcold" }
+; old: attributes #[[ATTR]] = { builtin allocsize(0) "memprof"="notcold" }
```

Comment:
unintentional test change? (the opt command doesn't run function-attrs)

Reply:
Thanks for reminding! Our PR causes this test failed and I'm not sure why.
Reached out to the owner of this test.

Reply:
Updated the linkage number as suggested by the owner and fixed this test!